tests/test_tokenization_common.py

```diff
@@ -156,28 +156,62 @@ def test_tokenizers_common_properties(self):
 
     def test_save_and_load_tokenizer(self):
```

Comment:
Maybe this test could be split in a few different tests later on? It's starting to get a bit thick.